index.html

```diff
@@ -56,6 +56,7 @@
       const cChainParams = {
           current: null,
           main: {
+              collateralInSats   : 10000*COIN,
```

Comment:
`COIN` is defined *after* the ChainParams, sooo... perhaps we move COIN before it? Otherwise, I assume this will end up as `NaN`.
It's more convenient than having to append `* COIN` to every collateral reference, so I don't mind moving it.

Reply:
Ah, dismiss this, my view was being strange and I couldn't see that `COIN` *was indeed* moved in to scope, nice, ignore. :smile: